src/include/migraphx/match/gelu_erf.hpp

```diff
@@ -38,11 +38,10 @@ struct gelu_erf_matcher
     F f;
     auto erf_fn() const
     {
-        return f("erf")(
-            used_once(),
-            arg(0)(used_once(),
-                   f("mul")(either_arg(0, 1)(none_of(has_value(M_SQRT1_2, 1e-3)).bind("x"),
-                                             has_value(M_SQRT1_2, 1e-3)))));
+        return f("erf")(used_once(),
+                        arg(0)(used_once(),
+                               f("div")(either_arg(0, 1)(none_of(has_value(1.414f, 1e-3)).bind("x"),
```

Comment:
`args` should be used instead of `either_arg(0, 1)` as division is not commutative. We want to match `x/sqrt(2)` and not `sqrt(2)/x`.